pkg/kyverno/test/test_command.go

```diff
@@ -397,9 +397,9 @@ func testCommandExecute(dirPath []string, fileName string, gitBranch string, tes
 				continue
 			}
 
-			if strings.Contains(file.Name(), fileName) || strings.Contains(file.Name(), "test.yaml") {
+			if file.Name() == "test.yaml" || file.Name() == fileName {
```

Comment:
Let's keep the filename condition first since that will short circuit more often.